make/images.mk

```diff
@@ -161,6 +173,24 @@ build-%:
 		ADDITIONAL_ARGS=\"$(ADDITIONAL_ARGS)\" \
 		ADDITIONAL_OVERRIDES=overrides/offline-fips.yaml,packer-$(call provider,$*)-offline-override.yaml$(if $(ADDITIONAL_OVERRIDES),$(COMMA)${ADDITIONAL_OVERRIDES})"
 
+.PHONY: %_offline-nvidia
+%_offline-nvidia:
+	$(MAKE) devkit.run WHAT="make packer-$(call provider,$*)-offline-override.yaml"
+	$(MAKE) os_distribution=$(call os_distro,$(call distro,$*)) \
+		os_distribution_os_release=$(call os_distro_os_release,$(call distro,$*)) \
+		os_distribution_major_minor_version=$(call version,$*) \
+		os_distribution_major_version=$(call major_version,$(call version,$*)) \
+		os_distribution_arch=x86_64 \
+		download-os-packages-bundle
+	$(MAKE) download-nvidia-runfile
+	$(MAKE) pip-packages-artifacts
+	$(MAKE) download-images-bundle
+	$(MAKE) devkit.run WHAT="make build-$* \
+		BUILD_DRY_RUN=${BUILD_DRY_RUN} \
+		VERBOSITY=$(VERBOSITY) \
+		ADDITIONAL_ARGS="--instance-type=g4dn.2xlarge$(if $(ADDITIONAL_ARGS),$(SPACE)$(ADDITIONAL_ARGS))" \
```

Comment:
this instance is what kaptain uses for their testing. we should do also test on this regularly. we've seen issues with this in the past 

https://jira.d2iq.com/browse/D2IQ-88483